### DISCUSSION TITLE
Manter duas versões de arquivo "ativas" no mesmo projeto

### DISCUSSION THREAD

**Discussion by RIldefonso** [balta-io/forum#502] [Tira Dúvidas] [answered]
Quando se usa APIs externas ou mesmo banco de dados, precisamos usar códigos, identificadores, ConnectionsStrings e senhas.

Se estamos compartilhando este código no Git, logicamente, não podemos deixar disponíveis tais informações, porém precisamos criar e/ou manter o arquivo de configuração para que os demais usuários e desenvolvedores saibam onde e como fazer adaptações para compilar e executar seu próprios "forks".

Como manter isso de forma fácil: ter o nosso arquivo válido em nossa máquina e fazer commits/pushs com o arquivo fake para ser adaptado por outros?

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @RIldefonso, aqui nesta aula eu explico:
https://balta.io/cursos/seguranca-apis-aspnet-jwt-bearer/3001M03A31

*Recomendação:*
Siga todas as carreiras, todos os cursos!
